packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1459,6 +1484,15 @@ class ListView extends BoxScrollView {
   /// the scroll position changes drastically.
   final double? itemExtent;
 
+  /// If non-null, forces the children to have the same extent as the given
+  /// widget in the scroll direction.
+  ///
+  /// Specifying an [prototypeItem] is more efficient than letting the children
+  /// determine their own extent because the scrolling machinery can make use of
+  /// the foreknowledge of the children's extent to save work, for example when
+  /// the scroll position changes drastically.
+  final Widget? prototypeItem;
```

Comment:
these docs should probably refer to SliverPrototypeExtentList

Reply:
(e.g. as a "see also", or a paragraph explaining how it works in terms of SliverPrototypeExtentList)